lib/wings/orm_converter.rb

```diff
@@ -53,7 +59,7 @@ def self.to_valkyrie_resource_class(klass:)
         include Wings::Works::FileSetValkyrieBehavior if klass.included_modules.include?(Hyrax::FileSetBehavior)
 
         # Based on Valkyrie implementation, we call Class.to_s to define the internal resource.
-        @internal_resource = klass.to_s
+        @internal_resource = (klass.try(:valkyrie_class) || klass).to_s
```

Comment:
Can we even get here if we have line 49 (`return klass.valkyrie_class if klass.respond_to?(:valkyrie_class)`)

Reply:
probably not. i think this got retained from some earlier refactoring. it's probably not necessary and i'll try backing it out.